src/states/node.rs

```diff
@@ -443,7 +443,14 @@ impl Node {
                    self,
                    peer_id,
                    pub_id);
-            self.find_tunnel_for_peer(peer_id, &pub_id);
+            if self.tunnels.tunnel_for(&peer_id).is_none() {
```

Comment:
I don't see the objection regarding https://maidsafe.slack.com/archives/routing/p1488466408009790 point 2?